src/main/java/com/sap/cloudfoundry/client/facade/domain/CloudServiceBinding.java

```diff
@@ -18,6 +18,9 @@ public abstract class CloudServiceBinding extends CloudEntity implements Derivab
 
     public abstract UUID getServiceInstanceGuid();
 
+    @Nullable
+    public abstract ServiceCredentialBindingOperation getServiceBindingOperation();
```

Comment:
I do not think it is a good idea to use the same class. The service credential binding does not have the "update" type. If a new type is added only for the service instance or only for the credential binding this might brake the deploy-service. Even in the cf API the last operation is show as a different entity for the service instance and the credential binding